README.md

```diff
@@ -2,127 +2,134 @@
 
 [![Build Status](https://travis-ci.org/tikv/client-rust.svg?branch=master)](https://travis-ci.org/tikv/client-rust)
 
-> Currently this crate is experimental and some portions (e.g. the Transactional API) are still in active development. You're encouraged to use this library for testing and to help us find problems!
-
 [Docs](https://www.tikv.dev/doc/rust-client/tikv_client/)
 
-This crate provides a clean, ready to use client for [TiKV](https://github.com/tikv/tikv), a
-distributed transactional Key-Value database written in Rust.
+This crate provides an easy-to-use client for [TiKV](https://github.com/tikv/tikv), a distributed, transactional key-value database written in Rust.
 
-With this crate you can easily connect to any TiKV deployment, interact with it, and mutate the data it contains. It uses async/await internally and exposes some `async fn` APIs as well.
+This crate lets you connect to a TiKV cluster and use either a transactional or raw (simple get/put style without transactional consistency guarantees) API to access and update your data.

```

Comment:
I prefer consistency, since ACID is not very well defined, and you could argue that SI is not ACID, etc. Also, in raw mode, operations are still atomic and durable.

Reply:
That makes sense. But "consistency" is ambiguous and barely says nothing about what guarantees we provide. How about "get/put style without transactions"?